src/ClientMessage.ts

```diff
@@ -1,206 +1,189 @@
-///<reference path="../typings/node/node.d.ts" />
```

Comment:
Are you sure? Because when I disable tslint and run `npm test` on this pull request, it fails with
```
src/ClientMessage.ts(3,23): error TS2307: Cannot find module 'long'.
src/ClientMessage.ts(8,20): error TS2304: Cannot find name 'Buffer'.
src/ClientMessage.ts(13,26): error TS2304: Cannot find name 'Buffer'.
src/ClientMessage.ts(24,24): error TS2304: Cannot find name 'Buffer'.
src/ClientMessage.ts(28,17): error TS2304: Cannot find name 'Buffer'.
src/ClientMessage.ts(125,25): error TS2304: Cannot find name 'Buffer'.
src/ClientMessage.ts(178,21): error TS2304: Cannot find name 'Buffer'.
src/ClientMessage.ts(181,26): error TS2304: Cannot find name 'Buffer'.
src/invocation/ClientConnection.ts(1,22): error TS2307: Cannot find module 'net'.
src/invocation/ClientConnection.ts(20,30): error TS2304: Cannot find name 'Buffer'.
src/invocation/ClientConnection.ts(34,20): error TS2304: Cannot find name 'Buffer'.
src/invocation/ClientConnectionManager.ts(32,61): error TS2304: Cannot find name 'Buffer'.
src/invocation/InvocationService.ts(19,37): error TS2304: Cannot find name 'Buffer'.
...
```
When I put `///<reference ...` comments back, it compiles.